ros_ws/src/autonomous/src/wall.cpp

```diff
@@ -0,0 +1,20 @@
+#include "wall.h"
+
+float Wall::getAngle()
+{
+    return atan((m_range1 * cos(std::abs(m_angle1 - m_angle2)) - m_range2) / m_range1 *
```

Comment:
Make sure that the used variables are initialized, by (for example) adding an ASSERT or initializing them in wall.h
Same thing goes for the two other methods of this class.